vuepress/docs/next/docs/getting-started/ent-bundle.md

```diff
@@ -104,6 +104,7 @@ See the [Build and Publish a Simple Bundle](../../tutorials/create/pb/publish-si
 - `ent bundle pack`:
    - The artifacts generated for micro frontends and microservices are stored in their respective component folders. File format depends on component type, e.g. a React micro frontend may result in HTML, JavaScript and CSS files.  
    - Once the artifacts are generated, a Docker image for each microservice is built using the Dockerfile located in the microservice's folder. If the Dockerfile is missing, the `pack` command exits with failure.
+   - The `pack` command executes either mvn package or npm run build, depending on the stack. These defaults can be overridden with a [custom command](https://developer.entando.com/v7.1/docs/curate/bundle-details.html#command-specification) section in the bundle descriptor `entando.json`.
```

Comment:
commands should be backticked here to avoid the reader thinking there's a missing article ("either an min package," etc.) in a command description and to clarify that they're looking at the commands themselves

custom command section sounds awkward; the contents of that section just boil down to commands, and interpreters work with commands on the lowest level. either custom command or custom commands, depending on if singular or plural is appropriate.

Reply:
commands backticked

it's a section in the entando.json and it can have multiple commands for the different commands